elements/pfe-accordion/test/pfe-accordion_test.js

```diff
@@ -16,9 +16,7 @@ suite('<pfe-accordion>', () => {
     const header = pfeAccordion.querySelector('pfe-accordion-header');
     const panel = pfeAccordion.querySelector('pfe-accordion-panel');
 
-    assert.equal(pfeAccordion.getAttribute('role'), 'tablist');
```

Comment:
It shouldn't matter anymore since we're not setting in the first place.

Reply:
We're not setting it in the component but that doesn't preclude someone else from having added role to their light DOM markup.  It would be good to validate that it gets removed.  Maybe add a role to the light DOM and validate it is stripped?

Reply:
Similar to my other comment below, I don't think we need to enforce the removal of a role attribute. If we did it for this component, we'd probably need to also do that for all of the other components.